<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sql_window</class>
 <widget class="QMainWindow" name="sql_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно БД</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1161</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>281</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>731</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSQLite_DB_Seer">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="openBD"/>
    <addaction name="exitTable"/>
   </widget>
   <addaction name="menuSQLite_DB_Seer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openBD">
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
  </action>
  <action name="exitTable">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
